docs/domains.rst

```diff
@@ -5,8 +5,10 @@ Domain Format
 
 The ``Domain`` defines the universe in which your bot operates.
 It specifies the ``intents``, ``entities``, ``slots``, and ``actions``
-your bot should know about. 
+your bot should know about.
 Optionally, it can also include ``templates`` for the things your bot can say.
+You can also define a list of intents, under ``intents_ignore_entities``, for
```

Comment:
I think we should think of a better name for `intents_ignore_entities`

Reply:
yeah i was also thinking that, any suggestions? i couldn't come up with anything good sounding that wasn't also reeeeally long

Reply:
yeah, me too :grinning: